envoyextensions/xdscommon/proxysupport.go

```diff
@@ -27,18 +79,28 @@ var EnvoyVersions = []string{
 // see: https://www.consul.io/docs/connect/proxies/envoy#supported-versions
 var UnsupportedEnvoyVersions = []string{}
 
-// GetMaxEnvoyMinorVersion grabs the first value in EnvoyVersions and strips the patch number off in order
-// to return the maximum supported Envoy minor version
+// GetMaxEnvoyMajorVersion grabs the first value in EnvoyVersions and strips the last number off in order
```

Comment:
Here and below, aligning these functions to Envoy's own "major/minor" non-semver naming conventions, similar to Consul and Go.